pulsar-broker/src/test/java/org/apache/pulsar/broker/intercept/CounterBrokerInterceptor.java

```diff
@@ -55,6 +55,7 @@
     int messageCount = 0;
     int messageDispatchCount = 0;
     int messageAckCount = 0;
+    int handleAckCount = 0;
```

Comment:
Why are these fields all `int`, not `long` ?

Reply:
The class is for local tests, I think int is ok